pkgs/development/tools/rust/cargo-bitbake/default.nix

```
@@ -0,0 +1,32 @@
+{ lib, pkg-config, rustPlatform, fetchFromGitHub, openssl }:
+
+rustPlatform.buildRustPackage rec {
+  powner = "meta-rust";
+  pname = "cargo-bitbake";
+  version = "0.3.15";
+
+  src = fetchFromGitHub {
+    owner = powner;
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "1ffjkwaqvmyz374azrv6gna19z2fcg82is2k2n2gm50isbxw2aa5";
+  };
+
+  nativeBuildInputs = [ pkg-config ];
+  buildInputs = [ openssl ];
+
+  cargoSha256 = "0mm6059wjh5p8923dwz55dpwi55gq2bcmpx7kn40pq5ppkiqjiw9";
+
+  meta = with lib; {
+    description =
+      "Cargo extension that can generate BitBake recipes utilizing the classes from meta-rust";
+    homepage = "https://github.com/${powner}/${pname}";
+    license = with licenses; [
+      mit
+      # or
+      asl20
+    ];
```

Opinionated change but re-licensing doesn't happen with any other release. Thus, a one-liner is fine :wink: 

```suggestion
    license = with licenses; [ mit asl20 ];
```